db/migrate/20200904040009_create_custom_profile_fields.rb

```diff
@@ -0,0 +1,18 @@
+class CreateCustomProfileFields < ActiveRecord::Migration[6.0]
+  def change
+    safety_assured do
```

Comment:
Since this is a brand new table adding the foreign key is harmless. If people prefer I can remove the `safety_assured` block and do it in two migrations instead.